index.bs

```diff
@@ -72,74 +73,83 @@ dictionary CSSStyleSheetInit {
 };
 </pre>
 
+<p class="note">
+Note that `[Constructor]` is defined in Web IDL to only be allowed on interfaces, not partial interfaces. 
+This is a known issue and we are going to move the definition to be on the <a for="/" interface>CSSStyleSheet</a> interface when we [merge this spec to CSSOM](https://github.com/w3c/csswg-drafts/issues/3433).
+</p>
 
 <dl>
 	<dt><dfn constructor for=CSSStyleSheet lt="CSSStyleSheet()|CSSStyleSheet(options)">CSSStyleSheet(|options|)</dfn></dt>
 	<dd>
 		When called, execute these steps:
 
-		1. Construct a new <a for="/" interface>CSSStyleSheet</a> object |sheet|,
-			with <a spec=cssom>location</a> set to the <a spec=html>base URL</a> of the {{Document}} for the {{Window}} where this constructor is called on,
-			no <a spec=cssom>parent CSS style sheet</a>,
-			no <a spec=cssom>owner node</a>,
-			no <a spec=cssom>owner CSS rule</a>,
-			and <a spec=cssom>title</a> set to the {{CSSStyleSheetInit/title}} attribute of |options|.
-		2.	Set |sheet|'s origin-clean flag.
-			Set |sheet|'s [=constructed flag=].
-			Set |sheet|'s [=constructor document=] to the {{Document}} for the {{Window}} where this constructor is called on.
-		3. If the {{CSSStyleSheetInit/media}} attribute of |options| is a string,
+		1. Construct a new <a for="/" interface>CSSStyleSheet</a> object |sheet| with the following properties:
```

Comment:
Why are you using an explicit `<a>` here? `{{CSSStyleSheet}}` works fine and is more idiomatic.

Reply:
It's complaining for ambiguity if I do that:
Local references:
spec:construct-stylesheets-1; type:constructor; for:CSSStyleSheet; text:CSSStyleSheet()
for-less references:
spec:cssom-1; type:interface; for:/; text:CSSStyleSheet
spec:cssom-1; type:interface; for:/; text:CSSStyleSheet

Reply:
Ah, indeed! You can specify that the `for` is `/` with just `{{/CSSStyleSheet}}`, then. ^_^

Reply:
oh, neat!